### PR TITLE
[TASK] Refactor: bindable members up top (RT-3011) 	-Refactored for balance, contacts and history pages

### PR DESCRIPTION
**Pull request by mrajvanshy**

### CODE REVIEW

File: src/js/tabs/balance.js [hunk 1]
```diff
@@ -37,8 +37,31 @@ BalanceTab.prototype.angular = function (module)
...
+    /*** My Orders widget */
+    $scope.sort_options = {
+      current_pair_only: false,
+      sort_field: 'type',
+      reverse: false
+    };
+
+    $scope.validationPatternCurrency = /^[a-zA-Z]{3}/;
+
+    ///////////////////////////
```
Comment (vhpoet) [inline src/js/tabs/balance.js]: We don't need this comment.

File: src/js/tabs/history.js [hunk 2]
```diff
@@ -73,10 +69,13 @@ HistoryTab.prototype.angular = function (module) {
...
 
+    // History states
+    $scope.$watch('loadState.transactions',function(){
+      $scope.historyState = !$scope.loadState.transactions ? 'loading' : 'ready';
+    });
 
-    $scope.advanced_feature_switch = Options.advanced_feature_switch;
-
-    $scope.orderedTypes = ['sent','received','gateways','trades','orders','other'];
+    //$scope.typeUsage = [];
```
Comment (MatthewPhinney) [inline src/js/tabs/history.js]: Do we need these comments?

Comment (mrajvanshy) [inline src/js/tabs/history.js]: I didn't take out any code that was already there. I'm not sure if it'll be needed later. @vhpoet can I take out these commented out lines?

File: src/js/tabs/history.js [hunk 3]
```diff
@@ -23,29 +23,25 @@ HistoryTab.prototype.angular = function (module) {
...
-    //$scope.typeUsage = [];
-    //$scope.currencyUsage = [];
-
     // Currencies from history
     var historyCurrencies = [];
 
+    // Latest transaction
+    var latest;
+
+    $scope.orderedTypes = ['sent','received','gateways','trades','orders','other'];
```
Comment (jublo) [inline src/js/tabs/history.js]: Can you add spaces after the commas, please?

File: src/js/tabs/history.js (hunk 2 above)
Comment (vhpoet) [inline src/js/tabs/history.js]: @mrajvanshy it's ok, you can leave it for now. This code is gonna change.

**Comment by mrajvanshy** [comment]
@jublo fixed.

**Comment by MatthewPhinney** [comment]
LGTM

**Pull request merged**